test/example_tests/libclang_example_test.dart

```diff
@@ -43,9 +43,25 @@ ${strings.enums}:
   ${strings.include}:
     - 'CXTypeKind'
     - 'CXGlobalOptFlags'
-${strings.typedefmap}:
-    'size_t': 'IntPtr'
-    'time_t': 'Int64'
+${strings.libraryImports}:
+  custom_import: 'custom_import.dart'
+${strings.typeMap}:
+  'typedef size_t':
```

Comment:
why is `typedef ` part of the key?

Reply:
I wanted to make `type-map` a generic way to override any import whatsoever. That means we need to be able to target structs, unions, typedefs and finally native types. Since C allows these to be named exactly the same as they are under different namespace, we need to follow suit.

The typemap in key signifies that only typedefs are to be matched. This has been documented in the configurations section of readme.

An alternative way could be to add a nested key for namespace.

Reply:
> An alternative way to implement this could be to add a nested key for namespace.

This feels cleaner to me. It also prevents people from having to repeat the namespace every definition.

Reply:
So do you mean something like this -

```yaml
type-map:
  typedef:
    'size_t':
      lib: pkg_ffi
      c-type: SIze_t
      dart-type: int
```

OR

```yaml
type-map:
  'size_t:
    lib: pkg_ffi
    c-type: Size_t
    dart-type: int
    namespace: typedef
```

Reply:
The first one. 👍

Reply:
Done. Also updated readme.